### ISSUE TITLE
Google Error Reported

### ISSUE THREAD

**Issue by adrian29630** [Automattic/amp-wp#183]
I have added the plug in and now added /amp/ to the "pretty" permalinks but am getting this error from Google in webmaster tools:-
Prohibited attributes:	type
AMP URL:	http://holdensothebysrealty.com/blog/welcome-michelle-matthews/amp/amp/
Canonical URL:	http://holdensothebysrealty.com/blog/welcome-michelle-matthews/amp/
Clicking through to the link brings up a 404 page not found result on both the above links. as it seems so far Google has only just begun to crawl the site I suspect I will get the same result on all posts.

I have the most up to date WordPress release 4.4 and the most up to date version of the plugin via WordPress plugins page.

Any advice as to the cause of the above appreciated.

Thanks

Adrian

**Comment by bph** [comment]
What happens when you use: 
http://holdensothebysrealty.com/welcome-michelle-matthews/amp/
That'w what I found in your canonical: 
<link rel="canonical" href="http://holdensothebysrealty.com/welcome-michelle-matthews/amp/" />

**Comment by adrian29630** [comment]
Hi Birgit

Strangly it now goes to the correct page whereas when I first viewed the error report it went to a 404 page not found. Perhaps whatever Was wrong has now corrected itself but I could not understand why it Was also showing the URL with amp displaying twice.

I will check again in webmaster tools and see if it no longer shows an error.

Thanks for responding.

Kind regards 

Adrian
Sent via BlackBerry® device.

-----Original Message-----

**Comment by adrian29630** [comment]
Hi Birgit

As a follow up I have checked again today and Google are now reporting the
same error on a number of further pages as below:-

Prohibited attributes:type
AMP URL:
http://holdensothebysrealty.com/dominican-republic-an-eclectic-mix-of-adventure/amp/amp/
Canonical URL:
http://holdensothebysrealty.com/dominican-republic-an-eclectic-mix-of-adventure/amp/
Last detected:2/11/16

I do not understand why the double /amp/amp/ is appearing as it is not
added in that form on WordPress Permalinks, only once, as instructed and
the canonical URL is correct. However, both these links work but it seems
as though the /amp/amp/ produces what would or should be the AMP page as it
is different to the /AMP/ link which just shows the page as it appears on a
desktop computer.

Any suggestions welcome

Thanks

Adrian

**Comment by mjangda** [comment]
The double `/amp/amp/` is weird. Might be something wrong with how your permalinks are set up. Can you share what your permalink structure looks like in `Settings > Permalinks`? Are you doing anything special with permalinks like a plugin to modify them?

**Comment by adrian29630** [comment]
Hi Mahammad

The permalink is holdensothebysrealty.com/%postname%/amp/  I am not using
anything special or plugins to modify the permalink structure.

Hope this helps.  I would use the standard post name option in permalinks (
http://holdensothebysrealty.com/sample-post/) but you can't add /amp/ to
this and I tried copying this to the custom structure but for some reason
it does not seem to work correctly with or without the /amp/ added.

Kind regards

Adrian

**Comment by adrian29630** [comment]
As a follow-up using Google's testing tool brings up the following:-


   - BlogPosting


   - mainEntityOfPage:

   http://holdensothebysrealty.com/doing-it-all-in-the-dominican-republic/amp/
   - headline:
   Doing it all in the Dominican Republic
   - datePublished:
   2015-07-03T16:12:44+00:00
   - dateModified:
   2015-07-03T16:12:44+00:00
   - publisher [Organization]:
   - logo:
   missing and required
   - name:
   missing and required
   - author [Person]:
   - name:
   Admin
   - image [ImageObject]:
   - url:

   http://holdensothebysrealty.com/wp-content/uploads/2015/07/dominican-republic-spa.jpg.size_.xxlarge.letterbox.jpg
   - width:
   545
   - height:
   365
   - The attribute *image.width* has an invalid value.

Not sure if this helps or not.

Kind regards

Adrian

**Comment by mjangda** [comment]
> The permalink is holdensothebysrealty.com/%postname%/amp/

You don't need `/amp/` in the permalink structure. It should just be `holdensothebysrealty.com/%postname%/`.

**Comment by adrian29630** [comment]
Hi Mohammad

Thank you. It seems I misinterpreted the plugin instructions, which to me
implied that you do:-

With the plugin active, all posts on your site will have dynamically
generated AMP-compatible versions, *accessible by appending /amp/ to the
end your permalinks.* (If you do not have pretty permalinks enabled,* you
can do the same thing by appending ?amp=1*.)

Assuming you are saying that the plugin adds /amp/ automatically by being
made active it might be clearer to say that rather than indicating that it
is necessary to append /amp/ if pretty permalinks are enabled or the
alternative
if not.  Or maybe it is just how I read the instructions!

Anyway I have removed the /amp/ from the permalink so will wait and see if
that has corrected the situation and google can display the AMP posts
correctly.

Thanks again

Adrian
